### ISSUE TITLE
Improve Semaphore settings

### ISSUE THREAD

**Issue by DenBond7** [FlowCrypt/flowcrypt-android#1078]
@DenBond7 Semaphore fails, likely due to some timeout
```
com.flowcrypt.email.ui.activity.ImportPrivateKeyActivityFromSettingsTest > testImportKeyFromBackup[ci-test-pixel-x86-api30(AVD) - 11] FAILED 24:19
        java.lang.RuntimeException: Waited for the root of the view hierarchy to have window focus and not request layout for 10 seconds. If you specified a non default root matcher, it may be picking a root that never takes focus. Root: 24:19
        Root{application-window-token=android.view.ViewRootImpl$W@4804303, window-token=android.view.ViewRootImpl$W@4804303, has-window-focus=false, layout-params-type=1, layout-params-string={(0,0)(fillxfill) sim={forwardNavigation} ty=BASE_APPLICATION wanim=0x10302fe
```
Please take a look.
I have already restarted semaphore few times and all times it failed on the one of the "Instrumentation tests" - each time on different one. Sometimes few instrumentation tests stopped in the middle. Maybe something can be done about this? Like increasing some timeouts.

_Originally posted by @IvanPizhenko in https://github.com/FlowCrypt/flowcrypt-android/issues/1077#issuecomment-797057636_

**Comment by tomholub** [comment]
I think Ivan ended up increasing the timeout in that PR which made it pass

**Comment by DenBond7** [comment]
We still have some problems:

1. Sometimes an emulator can't start or our script doesn't see that it was started
2. I think we should improve the performance. It takes too long time
3. I still can't guarantee 100% of the tests reproducing. On my PC in 99% of tests have the same result but on CI I often see issues

**Comment by IvanPizhenko** [comment]
@tomholub @DenBond7 Yes, I've ended up with increasing test run time limit from 30 min to 40 min, and that worked.

**Comment by DenBond7** [comment]
Closed via #1083
